libs/net/entities/User.cs

```diff
@@ -86,6 +86,12 @@ public class User : AuditColumns
     [Column("is_system_account")]
     public bool IsSystemAccount { get; set; }
 
+    /// <summary>
+    /// get/set - Whether the user turns off notifications.
+    /// </summary>
+    [Column("is_vacation_mode")]
+    public bool IsVacationMode {get; set; }
```

Comment:
Do not add this property to the root object. Add it to the user preferences.

Reply:
Added to preferences.